xapian-core/cluster/cluster.cc

```diff
@@ -296,9 +296,9 @@ Point::Point(const TermListGroup &tlg, const Document &document_)
 	string term = *it;
 	double termfreq = tlg.get_termfreq(term);
 
-	/// If term frequency of the term is lesser than or equal
-	/// to 1 or wdf is lesser than 1, we do not consider this
-	/// term for document vector calculations
+	// If the term doesn't exist within the document or
+	// exists in only one document within the MSet, then the
+	// terms are not used for document vector calculations
```

Comment:
I'm not sure of how to write this as a comment. But I had checked for which terms show a wdf of zero. I created a database locally and tried running KMeans on it. The terms I found were these:
`D20100330 Etxt I* J M201003 Orichhiey P/business Ttext/plain U/business/business_302.txt Y2010 `

I'm not sure of what they are called within Xapian but they seem like metadata terms which store the filepath, filename, etc. which I assume shouldn't be useful for clustering, because they aren't originally a part of the text document.
Could we frame this comment as :
`If the term exists in only one document within the MSet, or if it is a filter term or metadata term, then these terms are not used for document vector calculations`